lib/kbdctl/shortcut_keys.go

```diff
@@ -17,9 +17,7 @@ func SetupShortcutKeys(theatre *theatre.Theatre, ws *windowsink.WindowSink) {
 func keyCallback(theatre *theatre.Theatre, stageName string) func(w *glfw.Window, key glfw.Key, scancode int, action glfw.Action, mods glfw.ModifierKey) {
 	scenes := slices.Sorted(maps.Keys(theatre.Scenes))
 	names := make([]string, len(theatre.Scenes))
-	for i, n := range scenes {
-		names[i] = n
-	}
+	copy(scenes, names)
```

Comment:
Ah! I felt there was a more idiomatic way to do this but the docs did not help me.